src/sparkx/Lattice3D.py

```diff
@@ -1004,8 +1004,14 @@ def add_particle_data(self, particle_data, sigma, quantity, add = False):
             else:
                 raise ValueError("Unknown quantity for lattice.");
 
-            # Calculate the Gaussian kernel centered at (x, y, z)
-            kernel = multivariate_normal([x, y, z], cov=sigma**2 * np.eye(3))
+            if(kernel == "gaussian"):
+                # Calculate the Gaussian kernel centered at (x, y, z)
+                kernel_value = multivariate_normal([x, y, z], cov=sigma**2 * np.eye(3))
```

Comment:
I just did some research about the `multivariate_normal` function and found out that it seems to give us a random number sampled from the specified distribution. This is not what we need in any of the cases here. Shouldn't we just define a 3D Gaussian function equipped with a mean and variance?

Just to make it more clear, we need the value of the Gaussian distribution at the given point and not one random realization of it.

Reply:
We are using the PDF of the Gaussian distribution. The PDF is not random (see the plot at the bottom of the documentation here  https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.multivariate_normal.html)

Reply:
Ok, that's great.